### DISCUSSION TITLE
Tasmota TH (Thai) language support

### DISCUSSION THREAD

**Discussion by ziotprojects** [arendst/Tasmota#10292] [Q&A] [answered]
I would like to modify all messages under Tasmota software to support Thai language (LANGUAGE LCID 1054), please advice.

**Comment by jziolkowski** [comment]
Fork the repo, copy the en_GB file from i18n directory. Change the name to th_TH and translate as you see fit. Then, after it's done, make a pull request and wait for the file to be merged

**Comment by ascillato** [comment, marked as answer]
Duplicated Question. Please, check the answers given in https://github.com/arendst/Tasmota/discussions/10128
